src/System.Net.HttpListener/tests/GetContextHelper.cs

```diff
@@ -59,7 +59,7 @@ public class CustomAsyncResult : IAsyncResult
     public static class Helpers
     {
         public static bool IsWindowsImplementation { get; } = PlatformDetection.IsWindows && PlatformDetection.IsNotOneCoreUAP;
-        public static bool IsNotWindowsImplementation => !IsWindowsImplementation;
+        public static bool IsNotWindowsImplementation => !IsWindowsImplementation && PlatformDetection.IsNotOneCoreUAP;
```

Comment:
Though, looking at it again, I see a ton of tests disabled for PlatformDetection.IsNotOneCoreUAP, and this IsWindowsImplementation is factoring that in.  My change may end up breaking that, but we don't have any CI testing for UAP, do we?